ksim/task/rl.py

```diff
@@ -894,7 +894,9 @@ def get_termination_metrics(self, trajectories: Trajectory) -> dict[str, Array]:
         Args:
             trajectories: The trajectories to get the termination metrics for.
         """
-        num_terms = trajectories.done.sum(-1, dtype=trajectories.action.dtype) + 1
+        last_dones = jax.tree_map(lambda x: x[-1], trajectories.done)
```

Comment:
probably just do this
```suggestion
        last_dones = trajectories.done[-1]
```

Reply:
Ah right, it's structured